app/components/channel_list/loading_error/index.tsx

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import React from 'react';
+import {Text, View} from 'react-native';
+
+import CompassIcon from '@app/components/compass_icon';
+import TouchableWithFeedback from '@app/components/touchable_with_feedback';
+import {useTheme} from '@app/context/theme';
+import {buttonBackgroundStyle, buttonTextStyle} from '@app/utils/buttonStyles';
+import {typography} from '@app/utils/typography';
+import {changeOpacity, makeStyleSheetFromTheme} from '@utils/theme';
+
+const getStyleSheet = makeStyleSheetFromTheme((theme: Theme) => ({
+    container: {
+        flex: 1,
+        alignItems: 'center',
+        justifyContent: 'center',
+        padding: 20,
+    },
+    iconWrapper: {
+        height: 120,
+        width: 120,
+        backgroundColor: changeOpacity(theme.sidebarText, 0.08),
+        borderRadius: 60,
+        justifyContent: 'center',
+        alignItems: 'center',
+    },
+    icon: {
+        fontSize: 72,
+        lineHeight: 72,
+        color: changeOpacity(theme.sidebarText, 0.48),
+    },
+    header: {
+        color: theme.sidebarHeaderTextColor,
+        marginTop: 20,
+        textAlign: 'center',
+    },
+    body: {
+        color: theme.sidebarText,
+        textAlign: 'center',
+        marginTop: 4,
+    },
+}));
+
+const LoadingError = () => {
```

Comment:
I'm guessing this is going to be a more generic component so it can be reused, yes ?